regression/pages/studio/utils.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utility functions for studio page objects.
+"""
+from opaque_keys.edx.locator import CourseLocator
+
+
+def get_course_key(course_info, module_store='split'):
+    """
+    Returns the course key based upon course info passed.
+    Course key depends upon the module store passed. The default
```

Comment:
This docstring confuses me a bit. Is the course key different if it is split vs draft?